3p/ampcontext.js

```diff
@@ -312,8 +313,22 @@ export class AbstractAmpContext {
       this.setupMetadata_(this.win_.name);
     }
   }
-}
 
+  /**
+   * Send 3p error to parent iframe
+   * @private
+   */
+  report3pError_() {
+    this.win_.onerror = function(message) {
+      if (message) {
+        this.client_.sendMessage(MessageType.USER_ERROR, dict({
```

Comment:
the code here won't work. `this` changes in function. You can either use `() => {}` or bind `this` to the function.